[Notification Triggers article] Revive Notification Triggers (#3936)
GoogleChrome/web.dev: src/site/content/en/blog/notification-triggers/index.md

  The Notification Triggers API allows developers to schedule local notifications that don't require
  a network connection, which makes them ideal for use cases like calendar apps.
date: 2019-10-24

> updated: 2020-09-24

Author: ```suggestion
updated: 2020-09-25
```

| -------------------------------------------- | ---------------------------- |
| 1. Create explainer                          | [Complete][explainer]        |
| 2. Create initial draft of specification     | Not started                  |
| 3. Gather feedback and iterate on design.    | [In progress](#feedback)     |

> | **4. Origin trial**                          | [In progress][ot]            |

Author: ```suggestion
| **4. Origin trial**                          | **[In progress][ot]**        |
```

| 5. Launch                                    | Not started                  |

## How to use notification triggers {: #use }

### Enabling via chrome://flags

To experiment with the Notification Triggers API locally, without an origin trial token, enable the
`#enable-experimental-web-platform-features` flag in `chrome://flags`.

### Enabling support during the origin trial phase

Starting in Chrome&nbsp;86, the Notification Triggers API will be available
as an origin trial in Chrome. The origin trial is expected to end in
Chrome&nbsp;88 (February&nbsp;24, 2021).

{% Aside %}
  An earlier origin trial for the feature, which gave developers a first chance

>   to try out the proposed API, ran from Chrome&nbsp;80 to&nbsp;83
>   (see the summary of the
>   [obtained feedback](https://docs.google.com/document/d/1Nl1emEqxjTzPLNIAPiS26Vtq3mBdNyCxfMY6QwaD45s/edit)).

Author: ```suggestion
  to try out the proposed API, ran from Chrome&nbsp;80 to&nbsp;83.
  You can read the summary of the
  [obtained feedback](https://docs.google.com/document/d/1Nl1emEqxjTzPLNIAPiS26Vtq3mBdNyCxfMY6QwaD45s/edit))
  so far.
```

| -------------------------------------------- | ---------------------------- |
| 1. Create explainer                          | [Complete][explainer]        |
| 2. Create initial draft of specification     | Not started                  |
| 3. Gather feedback and iterate on design.    | [In progress](#feedback)     |
| **4. Origin trial**                          | **[In progress][ot]**        |
| 5. Launch                                    | Not started                  |

## How to use notification triggers {: #use }

### Enabling via chrome://flags

To experiment with the Notification Triggers API locally, without an origin trial token, enable the
`#enable-experimental-web-platform-features` flag in `chrome://flags`.

### Enabling support during the origin trial phase

Starting in Chrome&nbsp;86, the Notification Triggers API will be available
as an origin trial in Chrome. The origin trial is expected to end in
Chrome&nbsp;88 (February&nbsp;24, 2021).

{% Aside %}
  An earlier origin trial for the feature, which gave developers a first chance
  to try out the proposed API, ran from Chrome&nbsp;80 to&nbsp;83.
  You can read the summary of the

>   [obtained feedback](https://docs.google.com/document/d/1Nl1emEqxjTzPLNIAPiS26Vtq3mBdNyCxfMY6QwaD45s/edit))

Reviewer: ```suggestion
  [feedback obtained](https://docs.google.com/document/d/1Nl1emEqxjTzPLNIAPiS26Vtq3mBdNyCxfMY6QwaD45s/edit))
```
